cli/src/cmd/bind.rs

```diff
@@ -0,0 +1,184 @@
+use crate::cmd::Cmd;
+
+use clap::{Parser, ValueHint};
+use ethers::contract::MultiAbigen;
+use foundry_config::{
+    figment::{
+        self,
+        error::Kind::InvalidType,
+        value::{Dict, Map, Value},
+        Metadata, Profile, Provider,
+    },
+    impl_figment_convert, Config,
+};
+use serde::Serialize;
+use std::{fs, path::PathBuf};
+
+impl_figment_convert!(BindArgs);
+
+#[derive(Debug, Clone, Parser, Serialize)]
+pub struct BindArgs {
+    #[clap(
+        help = "The project's root path. By default, this is the root directory of the current Git repository or the current working directory if it is not part of a Git repository",
```

Comment:
Should this instead be "foundry.toml" > .git > cwd? Think there's a helper function somewhere @mattsse ?

Reply:
I copied this from another arg 😅